src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -328,7 +328,7 @@ public function get_result_messages( $queue ) {
 				_n( '%1$d new event category was created.', '%1$d new event categories were created.', $queue->activity->count( 'category', 'created' ), 'the-events-calendar' ),
 				$queue->activity->count( 'category', 'created' )
 			) .
-			' <a href="' . admin_url( 'edit.php?post_type=tribe_organizer' ) . '">' .
+			' <a href="' . admin_url( 'edit-tags.php?taxonomy=tribe_events_cat&post_type=tribe_events' ) . '">' .
```

Comment:
Should we check if the user is running WP 4.5+ and use the `term.php` page?

See: https://make.wordpress.org/core/2016/03/07/changes-to-the-term-edit-page-in-wordpress-4-5/